app/components/newMenuBar.js

```diff
@@ -0,0 +1,404 @@
+import { useState } from 'react';
+import { Navbar, Nav, Container, Col, Row, Offcanvas } from 'react-bootstrap';
+import styles from '../styles/Menubar.module.css';
+import { FirebaseAuthMenuButton } from './auth/firebase';
+
+const ArrowIcon = () => {
+  return (
+    <svg
+      width='14'
+      height='15'
+      viewBox='0 0 32 17'
+      fill='none'
+      xmlns='http://www.w3.org/2000/svg'
+    >
+      <path d='M1.5 1.5L15.5 15.5L31 1.5' stroke='black' stroke-width='2' />
+    </svg>
+  );
+};
+
+const navData = [
+  {
+    __component: 'menu.links',
+    id: 3,
+    label: 'Community',
```

Comment:
I think we solved this problem of "hard coded" nav structure?  And moving it to strapi ?  🤔

Reply:
yes i have followed @RonLek sir's approach i found it easier to implement and it can now be scaled to any depth.